src/deploy/NVA_build/FrontendLib.Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM noobaa-builder
+
+######################################################################
+# Layers:
+#   Title: npm install (using package.json)
+#   Size: ~ 805 MB
+#   Cache: Rebuild when there is new package.json or package-lock.json
+######################################################################
+COPY ./package*.json ./
+RUN npm install --production && \
+    npm cache clean --force
+
+######################################################################
+# Layers:
+#   Title: npm install the frontend
+#   Size: ~ 116 MB
+#   Cache: Rebuild when there is new package.json or package-lock.json 
+######################################################################
+COPY ./frontend/package*.json ./frontend/
+RUN cd frontend && \
+    npm install
+
+##############################################################
+# Layers:
+#   Title: Copying the code and Building the frontend
+#   Size: ~ 131 MB
```

Comment:
The Title should represent the building of the frontend dependencies.
The size is probably different, you can remove it (or update the value)

Reply:
done